### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by mariuszmmm**
Added pagination allows for:
- navigating to the first page,
- navigating to the previous page,
- navigating to the next page,
- navigating to the last page,
- displaying the current page number,
- displaying the total number of pages.
Logic for fetching data has been added to the pagination using "useState" temporarily, and it will be replaced with "Redux" store logic in the future.

### CODE REVIEW

**Review by Urbork — APPROVED** [review]
Works nice, very clean code

**Comment by Adrode** [comment]
Great job, nice use of hook

**Review by Adrode — APPROVED** [review]
Great job! Nice use of hook, everything looks simple (in a good sense) and fine

**Review by Adrode — CHANGES_REQUESTED** [review]
Please, change the colors formats from RGB to HEX. Use https://chir.ag/projects/name-that-color/  to apply names of the color to theme.js and then change color names in Pagination/styled.js

**Comment by mariuszmmm** [comment]
Color formats changed from RGB to HEX and added pagination function.

File: src/theme.js [hunk 1]
```diff
@@ -21,7 +21,15 @@ export const theme = {
...
     black: "#000000",
+    grey: "#E4E6F0",
+    darkerGrey: "#7E839A",
+    lightBlue: "#D6E4FF",
+    blue: "#0044CC",
+  },
+  breakpoint: {
+    tablet: 1023,
+    phone: 590,
+    mobileMax: "767px",
```
Comment (Adrode) [inline src/theme.js]: Good job!

**Review by Adrode — APPROVED** [review]
Now it looks perfect!

**Pull request merged**